code/datums/ruins/space.dm

```diff
@@ -254,61 +364,106 @@
+	description = "An experimental cloning lab snapped off from an ancient ship. The cloner model inside lacks many modern functionalities and security measures."		
+	cost = 2
 
 /datum/map_template/ruin/space/hilbertresearchfacility
 	id = "hilbert_facility"
 	suffix = "hilbertshoteltestingsite.dmm"
 	name = "Hilbert Research Facility"
-	description = "A research facility of great bluespace discoveries. Long since abandoned, willingly or not..."
+	description = "A research facility of great bluespace discoveries. Long since abandoned, willingly or not..."	
+	cost = 2
 
 /datum/map_template/ruin/space/clownplanet
 	id = "clownplanet"
 	suffix = "clownplanet.dmm"
 	name = "Clown Planet"
-	description = "Thought lost in 2552, this minor planet has recently been rediscovered."
+	description = "Thought lost in 2552, this minor planet has recently been rediscovered."		
+	cost = 2
 
 /datum/map_template/ruin/space/macspace
 	id = "fast_food"
 	suffix = "Fast_Food.dmm"
 	name = "Mac Space Restaurant"
 	description = "A fast food reataurant in space."
+	cost = 5
 
 /datum/map_template/ruin/space/scav_mining
 	id = "mining_asteroid"
 	suffix = "scav_mining.dmm"
 	name = "Mining asteroid"
-	description = "An abandoned mining operation on an asteroid that now has new ocupants that is not happy to se you"
+	description = "An abandoned mining operation on an asteroid that now has new ocupants that is not happy to se you"	
+	cost = 2
 
 /datum/map_template/ruin/space/power_puzzle
 	id = "power_puzzle"
 	suffix = "power_puzzle.dmm"
 	name = "Power Puzzle"
 	description = "an abandoned secure storage location. there is no power left in the batteries and the former ocupants locked it pretty tight before leaving.\
-	You will have to power areas to raise the bolts on the doors. look out for secrets."
\ No newline at end of file
+	You will have to power areas to raise the bolts on the doors. look out for secrets."
+	cost = 4
+
+/datum/map_template/ruin/space/spacearcade
+	id = "spacearcade"
+	suffix = "spacearcade.dmm"
+	name = "Space Arcade"
+	description = "Come here to have fun! We even have the best arcade game in the Universe, Dance Dance Revolution!"	
+	cost = 2
+
+/datum/spacedock13/ruin/space/spacedock
+	id = "spacearcade"
+	suffix = "spacedock13.dmm"
+	name = "Shuttle Construction"
+	description = "A shuttle building station that seems to have been forgotten, or floated off into space."	
+	cost = 2
+
+/datum/spacedock13/ruin/space/spacehive
+	id = "spacehive"
+	suffix = "spacehive.dmm"
+	name = "Space Hive"
+	description = "A hydroponics experiment gone horribly, horribly wrong!"
+	cost = 1
+
+/datum/spacedock13/ruin/space/swarmerstation
+	id = "swarmerstation13"
+	suffix = "swarmerstation13.dmm"
+	name = "Swarmer Station"
+	description = "Shat is left of a syndicate radio station, after swarmers infiltrated it."	
+	cost = 2
+
+/datum/spacedock13/ruin/space/refference
```

Comment:
```suggestion
/datum/map_template/ruin/space/spacedock
	id = "spacearcade"
	suffix = "spacedock13.dmm"
	name = "Shuttle Construction"
	description = "A shuttle building station that seems to have been forgotten, or floated off into space."	
	cost = 2

/datum/map_template/ruin/space/spacehive
	id = "spacehive"
	suffix = "spacehive.dmm"
	name = "Space Hive"
	description = "A hydroponics experiment gone horribly, horribly wrong!"
	cost = 1

/datum/map_template/ruin/space/swarmerstation
	id = "swarmerstation13"
	suffix = "swarmerstation13.dmm"
	name = "Swarmer Station"
	description = "Shat is left of a syndicate radio station, after swarmers infiltrated it."	
	cost = 2

/datum/map_template/ruin/space/refference
```

Come on, I know you are better than *this*.

Reply:
:)